query/language/define.feature

```diff
@@ -2457,116 +2390,116 @@ Feature: TypeQL Define Query
   ########################
 
   Scenario: a type cannot be a subtype of itself
-    Then typeql define; throws exception
+    Then typeql define; fails
       """
       define dog sub dog;
       """
 
 
   Scenario: a cyclic type hierarchy is not allowed
-    Then typeql define; throws exception
-      """
-      define
-      giant sub person;
-      green-giant sub giant;
-      person sub green-giant;
-      """
-
-
-  Scenario: two attribute types can own each other in a cycle
-    Given typeql define
-      """
-      define
-      nickname sub attribute, value string, owns surname, owns middlename;
-      surname sub attribute, value string, owns nickname;
-      middlename sub attribute, value string, owns firstname;
-      firstname sub attribute, value string, owns surname;
-      """
-    Then get answers of typeql get
-      """
-      match $a sub attribute, owns $b; $b sub attribute, owns $a; get;
-      """
-    Then uniquely identify answer concepts
-      | a              | b              |
-      | label:nickname | label:surname  |
-      | label:surname  | label:nickname |
-    Then get answers of typeql get
-      """
-      match $a owns $b; $b owns $a; get;
-      """
-    Then uniquely identify answer concepts
-      | a              | b              |
-      | label:nickname | label:surname  |
-      | label:surname  | label:nickname |
-
-  Scenario: many attribute types can own each other in a big cycle
-    Given typeql define
-      """
-      define
-      nickname sub attribute, value string, owns surname, owns middlename;
-      surname sub attribute, value string, owns nickname;
-      middlename sub attribute, value string, owns firstname;
-      firstname sub attribute, value string, owns surname;
-      """
-    Then get answers of typeql get
-      """
-      match
-      $a sub attribute, owns $b;
-      $b sub attribute, owns $c;
-      $c sub attribute, owns $d;
-      $d sub attribute, owns $a;
-      get;
-      """
-    Then uniquely identify answer concepts
-      | a                | b                | c                | d                |
-      | label:middlename | label:firstname  | label:surname    | label:nickname   |
-      | label:firstname  | label:surname    | label:nickname   | label:middlename |
-      | label:surname    | label:nickname   | label:middlename | label:firstname  |
-      | label:surname    | label:nickname   | label:surname    | label:nickname   |
-      | label:nickname   | label:middlename | label:firstname  | label:surname    |
-      | label:nickname   | label:surname    | label:nickname   | label:surname    |
+    Then typeql define; fails
+      """
+      define
+      entity giant sub person;
+      entity green-giant sub giant;
+      entity person sub green-giant;
+      """
+
+
+#  Scenario: two attribute types can own each other in a cycle
+#    Given typeql define
```

Comment:
Same as with attributes playing roles, scenarios dealing with attributes owning attributes should go as well.